CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ add_executable(unit
 )
 
 target_include_directories(unit PRIVATE "test" "src" "test/thirdparty")
+target_compile_options(unit PUBLIC -Werror -Wall -Wextra)
```

Comment:
These flags will only work with Clang or GCC. Please do not hard-code them here.